### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Coffee2CodeNL**
Added Gitter.im chat for support.

As discussed in #5394

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -171,10 +171,11 @@ For further details, consult the [wiki](https://github.com/ReactiveX/RxJava/wiki
 
 ## Communication
 
+- [Gitter.im](https://gitter.im/ReactiveX/RxJava)
```
Comment (akarnokd) [inline README.md]: Why regrouping? Actually I prefer StackOverflow because questions can be isolated much better. The gitter channels usually invoke chaos by talking over each other and hard to follow who reacted to what.

Comment (Coffee2CodeNL) [reply README.md]: Realtime -> "Slightly Slower" communication -> Social Media was my idea. 
Unless you want it sorted by preferred communication channels, then i'll update it to your liking.

Comment (akarnokd) [reply README.md]: Please add this to the bottom without changing the order of the existing list elements.
